src/draw/lv_image_buf.h

```diff
@@ -103,6 +104,7 @@ typedef struct {
     uint32_t h: 16;
     uint32_t stride: 16;        /*Number of bytes in a row*/
     uint32_t reserved_2: 16;    /*Reserved to be used later*/
+    uint32_t rotation : 16;     /*Image angle of rotation*/
```

Comment:
Same as before: rotation is meta info (EXIF data) in source image, LVGL does not process that info. 
For rotating images there is a transform feature in LVGL.